### PR TITLE
Domain publishing

### PR DESCRIPTION
**Pull request by awfuchs**
Adds an overview of domain publishing and enterprise extensions generally, and provides links to the relevant enterprise topics. Adds the following

- New [Domain publishing](https://deploy-preview-919--developer-chrome-com.netlify.app/docs/webstore/cws-dashboard-distribution/#private-visibility-domain-publish) heading on the distribution page
- New [Enterprise publishing options](https://deploy-preview-919--developer-chrome-com.netlify.app/docs/webstore/cws-enterprise/) page that summarizes topics and links to enterprise docs.

### CODE REVIEW

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+This option is only available to domain users when it has been enabled for the domain. Make sure that:
+
+* You are logged in using your domain identity, and not a public (gmail) or other non-domain identity.
+* Your administrator has configured domain publishing.
+
+**Why are my users not getting extension updates after I publish them?**
+
+It can take a while for Chrome to pull updates from the Chrome Web Store. Make sure your users are
+staying logged into Chrome with their domain identity for at least a few hours at a time.
```
Comment (dotproto) [inline site/en/docs/webstore/cws-enterprise/index.md:127]: Another potential cause of issue here is that policies can take a while to propagate. If they just updated a policy to add a new extension, they may need to wait for it to actually be picked up by Chrome. [This support article](https://support.google.com/chrome/a/answer/187202?hl=en#zippy=%2Cwindows) touches a little on propagation at the end of the "Windows" section. I'm not sure how deep we want to get into the troubleshooting, but users can see Chrome's current policies by visiting `chrome://policy`.

Additional lag can also be introduced by the CWS's rollout process. This is probably best covered in a new page dedicated to extension updates, but the short version is that Omaha (the service that backs the Chrome and CWS update process) has differential serving and rolls out updates to an extension's user base over time.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+**I published an extension to my domain but I don't see it on the Chrome Web Store.**
```
Comment (dotproto) [inline site/en/docs/webstore/cws-enterprise/index.md:129]: May also be worth expanding this item to note that updates roll out over a period (and linking to the appropriate doc once it exists).

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+There are ways to install enterprise extensions without using the Chrome Web Store, such as those
+described in this section.
+
+**Force install:** Administrators can set policies that automatically install extensions
+for users in the organization. This is typically used for extensions that support workflows and
+other core business operation.
+  
+See the [ExtensionInstallForcelist][ent-ext-admin-forcelist] documentation to learn more about
+this feature.
```
Comment (dotproto) [inline site/en/docs/webstore/cws-enterprise/index.md:110]: We should validate with Fletcher whether ExtensionInstallForceList, ExtensionSettings, or both are the prefered method of controlling this.

**Review by dotproto — CHANGES_REQUESTED** [review]
I've flagged this review as "request changes." I don't think any of my comments in this review are blocking, but I'd like to do another review before we merge.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+Picks, but appear in your private Chrome Web Store.
+
+The article [Create a Chrome app collection][ent-ext-admin-collections] provides instructions for
+administrators to add collections to your private Chrome Web Store.
+
+**Blocklist and allowlist:** These are options that your organization's administrators can
+use to explicitly control which extensions may or that may not be installed by members of the org.
+
+These and other enterprise extension controls are set by administrators using policies in the
+[extensions atomic policy group][ent-ext-admin-policies].
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:93]: Acknowledged.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  
+See the [ExtensionInstallForcelist][ent-ext-admin-forcelist] documentation to learn more about
+this feature.
+
+## Troubleshooting tips for domain extensions {: #troubleshooting }
+
+This section addresses a few common issues that extension developers may encounter when publishing
+to a domain.
+
+**I don't see the "Only to my domain" option under the private publishing option.**
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:117]: The linter objected to punctuation in headings.

File: site/en/docs/webstore/cws-enterprise/index.md (hunk 3 above)
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:110]: He did not comment on this in his review, while commenting on adjacent text.

File: site/en/docs/webstore/cws-dashboard-distribution/index.md [hunk 6]
```diff
@@ -54,6 +54,24 @@ manage:
...
+This enterprise feature is only available for Google Workspace domains where it has been enable by
+the domain administrator.
+
+{% endAside %}
+
+If domain publishing is enabled for your organization, an additional option appears in the
+distribution page of the developer console:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/SmrqQb3mLZaYj0O3Kdiv.png", alt="Screenshot showing
+the domain publishing option in the developer console", width="643", height="200", class="screenshot" %}
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:70]: Done.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+layout: "layouts/doc-post.njk"
+title: "Enterprise publishing options"
+#date: TODO
+#updated: TODO
+description: How to distribute  extensions to enterprise users
+---
+
+When you are in a Google Workspace organization, there are some additional options available for
+publishing extensions to your domain. This article summarizes these options, but they are detailed as
+part of the [Chrome enterprise documentation][ent-chrome-management].
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:11]: Points to a better resource now.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+this, you need to use a slightly different URL than that of the public store, which specifies your
+Google Workspace domain:
+
+* **Public Chrome Web Store:** &emsp;
+  `chrome.google.com/webstore/category/extensions`
+* **Private Chrome Web Store:** &emsp;
+  `chrome.google.com/webstore/a/example.com/category/extensions`
+
+The option that enables this is available to administrators at `Devices>Chrome>Apps & extensions>users
+& browsers>additional settings>chrome Web store permissions`:
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:44]: Fixed the formatting; created b/193527772 to track the validation need.

File: site/en/docs/webstore/cws-enterprise/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:123]: This is getting fairly involved, but it's a good point. Tracking as b/180055398.

File: site/en/docs/webstore/cws-enterprise/index.md (hunk 2 above)
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:125]: Added another bullet for this.

File: site/en/docs/webstore/cws-dashboard-distribution/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:70]: Looks like the double border is still there.

**Comment by dotproto** [comment]
@googlebot I consent

**Review by dotproto — CHANGES_REQUESTED** [review]
We may want to update the screenshots to use dummy data or redact information.

- The screenshot at the top of the `cws-dashboard-distribution` page appears to contain an extension's name and ID
- The screenshot in the "Private visibility: Trusted testers" section contains the publisher identifier in the top right corner
- The screenshot in the "Private visibility: Google Groups" section contains the publisher identifier in the top right corner and a list of Google Groups names and addresses

The "Setting the visibility" section of the `cws-dashboard-distribution` page contains a bulleted list that describes the top level visibility options, but does not currently indicate that the meaning of the "Private" option changes based on Google Workspace settings. For a normal Google account, "Private" corresponds to the option to publish to trusted testers.

Standard Google account options

Unfortunately I don't have a Google Workspace account handy, so I can't capture the options available for those users.

The screenshot in the "Private visibility: Google Groups" section of the `cws-dashboard-distribution` page appears to have the "Unlisted" radio button selected while also showing the dropdown for Google Group selection. This is a little confusing given that we're showing a private option in more detail. It's also worth noting that the presence of this field appears to depend on whether or not the account you're logged into is a member of a Google Workspace. If you're not Google Workspace user, this box is not shown unless you select Private, then select the "Only trusted testers…" option.

**Review by dotproto — APPROVED** [review]
Discussed this in today's sync. Nothing major remaining, so approving for merge.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+You also need to use a slightly different URL than that of the public store, which specifies your
+Google Workspace domain:
+
+* **Public Chrome Web Store:** &emsp;
+  `chrome.google.com/webstore/category/extensions`
+* **Private Chrome Web Store:** &emsp;
+  `chrome.google.com/webstore/a/example.com/category/extensions`
+
+The option that enables this is available to administrators at **Devices > Chrome > Apps &
+extensions > users & browsers > additional settings > chrome Web store permissions**:
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:45]: Converted to sentence case as discussed.

File: site/en/docs/webstore/cws-dashboard-distribution/index.md [hunk 10]
```diff
@@ -54,6 +54,24 @@ manage:
 Any members of the specified groups, along with your trusted testers, will be able to install the
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:54]: The publisher IDs and Google Group names are redacted. Remaining items rolled over to b/195010083.

File: site/en/docs/webstore/cws-enterprise/index.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+* **Force install** &mdash; Administrators can set policies that automatically and silently install
+  extensions for users in the organization. Once installed, users cannot disable or remove these
+  extensions.
+
+The following sections provide a brief overview of these features. See the cited enterprise
+documentation for each topic to learn about them in more detail.
+
+## Private Chrome Web Store for your organization {: #private-cws }
+
+Any Google Workspace organization can have its own private instance of Chrome Web Store. To access
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/cws-enterprise/index.md:33]: I agree that "the" should appear in this instance. I will work out heuristics for this later. Done.

**Pull request merged**